network-api/networkapi/buyersguide/templates/bg_base.html

```diff
@@ -5,10 +5,13 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta property="og:type" content="website" />
     {% block og-title %}
-      <meta property="og:title" content="privacy not included" />
+      <meta property="og:title" content="*Privacy Not Included: A Holiday Buyer’s Guide for Connected Products" />
     {% endblock %}
-    <meta property="og:description" content="This year, learn what tech comes with privacy included, using Mozilla's buyer's guide for connected gadgets." />
+    <meta property="og:description" content="70 connected products rated on their safety and security" />
```

Comment:
Should we make this dynamic? Like `content="{{ products.all | len }} connected products ..."`?